wolfcrypt/src/aes.c

```diff
@@ -463,6 +463,18 @@
         }
         CRYP_KeyInit(&AES_CRYP_KeyInitStructure);
 
+        /* set direction, key, and datatype */
+        AES_CRYP_InitStructure.CRYP_AlgoDir  = CRYP_AlgoDir_Decrypt;
+        AES_CRYP_InitStructure.CRYP_AlgoMode = CRYP_AlgoMode_AES_Key;
+        AES_CRYP_InitStructure.CRYP_DataType = CRYP_DataType_8b;
```

Comment:
Not sure that it matters, but the stdperihlib used CRYP_DataType_32b instead. I don't it actually matters since it should only affect input data, not the key as such.